metron-sensors/bro-plugin-kafka/README.md

```diff
@@ -94,6 +95,60 @@ event bro_init()
 }
 ```
 
+### Example 3
+
+As documented in [METRON-285](https://issues.apache.org/jira/browse/METRON-285) and [METRON-286](https://issues.apache.org/jira/browse/METRON-286), various components in Metron do not currently support IPv6.  Because of this, you may not want to send bro logs that contain IPv6 source or destination IPs into Metron.  In this example, we are assuming a somewhat standard bro configuration for sending logs into a Metron cluster, such that:
+ * Each type of bro log is sent to the `bro` topic, but is tagged with the appropriate log type (such as `http`, `dns`, or `conn`).  This is done by setting `topic_name` to `bro`, setting `$path` to an empty string (or leaving it unset), and by setting `tag_json` to true.
+ * The Kafka writer is set appropriately to send logs to the `bro` Kafka topic being used in your Metron cluster.  This requires that your `kafka_conf` and `$config` tables are appropriately configured.
+
+```
+@load Bro/Kafka/logs-to-kafka.bro
+redef Kafka::topic_name = "bro";
```

Comment:
Not sure I follow.  If `Kafka::logs_to_send` is empty why would `logs-to-kafka.bro` make its own filters?